prompterator/utils.py

```diff
@@ -362,3 +380,37 @@ def insert_hidden_html_marker(helper_element_id, target_streamlit_element=None):
 def format_traceback_for_markdown(text):
     text = re.sub(r" ", "&nbsp;", text)
     return re.sub(r"\n", "\n\n", text)
+
+
+def build_function_calling_tooling(json_schema):
```

Comment:
@gkaretka based on how this is done, it would be nice to get some small help section added to `README` as well